docs/how_to_start_developing_the_backend.md

```diff
@@ -0,0 +1 @@
+# How to start developing the backend
```

Comment:
@kome12 Yukari
please add here a how to get started, when you have time🙏

Reply:
@daiHash Will do!